frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/SelectEmbedTypePane.unit.spec.tsx

```diff
@@ -158,4 +164,16 @@ describe("SelectEmbedTypePane", () => {
       });
     });
   });
+
+  describe('"Compare options" button', () => {
```

Comment:
[Test this](https://www.notion.so/metabase/Streamline-embed-flow-when-embedding-features-are-off-13869354c90180b4bc02ca29ebf61ad2?pvs=4#14969354c90180b0872ef52aa27317e5)